pkgs/development/libraries/ensmallen/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, lib, armadillo, cmake, fetchFromGitHub, openmp ? null, enableOpenMP ? false }:
+
+stdenv.mkDerivation rec {
+  name = "ensmallen-${version}";
```

```suggestion
  pname = "ensmallen";
```

We've got this now for all of nixpkgs :)
It creates `name` automatically from `version` and `pname`.